src/app/domains/MediaLibrary/combined/MediaLibraryModal/MediaLibraryModal.template.js

```diff
@@ -54,43 +69,49 @@ function MediaLibraryModal(props) {
   }
   const modalStateChange = () => {
     setIsModalVisible(!isModalVisible)
+    onClick && onClick()
+    // setIsImageEditVisible(false)
   }
-  const onAddForm = () => {
-    setFormsList((prev) => [
-      ...prev,
-      {
-        title: 'new Title ',
-        subtitle: 'subtitle'
-      }
-    ])
-  }
+  // const onAddForm = () => {
```

Comment:
remove this lines it’s not necessary anymore

Reply:
done